defaults/main.yml

```diff
@@ -268,6 +268,7 @@ camac_python_pyenv_virtualenv_path: "{{ camac_python_pyenv_install_directory }}/
 
 # Url of the Camac Django API
 camac_python_url: "http://camac-ng.local/"
+camac_internal_url: "https://localhost"
```

Comment:
Are you sure https works on localhost?

Reply:
Why shouldn't it? This request will be handled by apache which has https configured

Reply:
I just wonder if the cert is valid for the `localhost` domain...

Reply:
You're absolutely right.. I have to configure the URL of the uwsgi service directly..